dist/calendar/CalendarMonth.js

```diff
@@ -65,7 +60,33 @@ var CalendarMonth = _react2['default'].createClass({
     highlightedRange: _react2['default'].PropTypes.object,
     onMonthChange: _react2['default'].PropTypes.func,
     onYearChange: _react2['default'].PropTypes.func,
-    value: _utilsCustomPropTypes2['default'].momentOrMomentRange
+    value: _utilsCustomPropTypes2['default'].momentOrMomentRange,
+    locale: _react2['default'].PropTypes.string
+  },
+
+  _setupLocale: function _setupLocale(locale) {
+    this.moment = (0, _moment2['default'])();
+
+    this.moment.locale(locale);
+
+    var lang = this.moment.localeData(locale);
+
+    this.WEEKDAYS = _immutable2['default'].List(lang._weekdays).zip(_immutable2['default'].List(lang._weekdaysShort));
```

Comment:
Hey
Its noot good to get just `lang._weekdays` or `lang._month`, its not always an array (e.g for `ru` locale https://github.com/moment/moment/blob/develop/locale/ru.js#L38)
Check https://github.com/moment/moment/blob/602718d67f06011cf6483387ad96989528f55597/src/lib/units/month.js#L65